### ISSUE TITLE
[FEATURE] Aktenaufruf über Desktop-Etikette sollte das Etikett als Filter in der Dokumentenansicht behalten

### ISSUE THREAD

**Issue by LazyGuy21No** [jlawyerorg/j-lawyer-org#1761]
Bis es einen guten Workflow gibt, wäre es schön, wenn bei einem Aktenaufruf über die Desktop-Etikette-Ansicht auf dem Desktop das Etikett über das aufgerufen wurde, als Filter in der Dokumentenansicht gesetzt bleiben würde, um das Dokument schneller wieder zu finden.

**Comment by j-dimension** [comment]
Klickt man im Desktop auf ein etikettiertes Dokument, so wird exakt dieses Dokument automatisch selektiert, wenn man navigiert. Einen Filter oder eine Suche braucht es dort nicht. 

Funktioniert dieses Verhalten bei Dir nicht?

**Comment by LazyGuy21No** [comment]
Doch, doch! Aber es gibt gibt in derselben Akte oft viele Dokumente mit demselben Etikett (versenden per beA). Die anderen mit demselben Etikette sehe ich erst, wenn ich das Etikett wieder über das Etikett-Icon neben der Suchleiste auswähle.

Ich komme ja sozusagen aus der Ansicht, die mir sagt, dass ich um dieses Etikette kümmere und würde diesen "Blick auf die Akte" gern behalten.

Es mag an meiner Schusseligkeit liegen, aber es ist mehr als nur einige, wenige Male passiert, dass ich die Dokumente in derselben Sache tröpfchenweise über das beA geschoben habe. Oder ich war in derselben Akten, um mir den Posteingang (mehrfach) anzuschauen.
